_common/qiskit/execute.py

```diff
@@ -33,9 +33,15 @@
 import numpy as np
 
 from datetime import datetime, timedelta
-from qiskit import transpile
-from qiskit_aer import Aer
+from qiskit import QuantumCircuit, transpile
 from qiskit.providers.jobstatus import JobStatus
+from qiskit.primitives import StatevectorSampler
```

Comment:
allows `StatevectorSampler` path for testing on the simulator by setting `backend_id =  "statevector_sampler"`.